docs/subsystems/typing-indicators.md

```diff
@@ -102,6 +102,10 @@ potential recipients of the message. The event type for that
 payload is `typing`. See the function `do_send_typing_notification`
 in `zerver/actions/typing.py` for more details.
 
+For channel typing notifications, the server also handles the logic
+for determining which users should receive the typing events based
+on channel subscribers and message visibility.
```

Comment:
> and message visibility

This shouldn't affect channel typing events. Relevant code:

https://github.com/zulip/zulip/blob/cabe0b2e0506a3d33cd0328f29089c53ba509124/zerver/actions/typing.py#L95-L99

Let's just drop the part "and message visibility".